python/cookiecutter-django.md

```diff
@@ -0,0 +1,546 @@
+# How to start a new Django project with Cookiecutter
+The aim of this guide is to help you to initialize your Django project from one of the most recognized open-source utilities for this area.
+
+## Install Cookiecutter
+[*Cookiecutter*](https://github.com/cookiecutter/cookiecutter) is a command-line tool that helps us to create projects from different templates, in our case, we will create a Django project.
+
+Firstly, we have to install that tool running `$ pip install cookiecutter`
+
+## Create a new project
+How we want to create a Django project, we need to use the [Cookiecutter Django](https://github.com/pydanny/cookiecutter-django) template.
```

Comment:
```suggestion
Since we want to create a Django project, we need to use the [Cookiecutter Django](https://github.com/pydanny/cookiecutter-django) template.
```